vipps-order-management-api.md

```diff
@@ -180,72 +78,208 @@ everyone sends it too. It can speed up any trouble-shooting quite a bit.
-one for point of sale (POS) integrations and one for web shops,
-the headers should be:
+The Order Management Api does **no validation if the order exist**. This means that the order management enrichment may be added before or after the payment is actually created. The preferred way is to add the Order Management Data simultaneously as you initiate the payment. 
 
-| Header                        | Example value for POS | Example value for webshop | Example value for Vending machines |
-| ----------------------------- | --------------------- | ------------------- | ------------------- |
-| `Merchant-Serial-Number`      | `123456`              | `123456`            | `123456`            |
-| `Vipps-System-Name`           | `acme`                | `acme`              | `acme`              |
-| `Vipps-System-Version`        | `1.7`                 | `2.6`               | `2.6`               |
-| `Vipps-System-Plugin-Name`    | `acme-pos`            | `acme-webshop`      | `acme-vending`      |
-| `Vipps-System-Plugin-Version` | `3.2`                 | `4.3`               | `4.3`               |
+As the same OrderId can be used for both a Recurring charge and a Ecom payment, you need to supply which Vipps Product is being used by setting the appropriate `paymentType`
 
-**Important:** Please use self-explanatory, human readable and reasonably short
-values that uniquely identify the system (and plugin).
+## Basic flow
+1. Initiate a Vipps eCom or recurring payment
+    - `POST:/ecomm/v2/payments` 
+2. Save the **orderId** you used when initiating
+3. Add an image (optional)
+    - `POST:/v1/images` 
+4. Save the imageId
+5. Add a category with link and image
+    - `/v2/{paymentType}/categories/{orderId}`
+6. Add a receipt 
+    - `/v2/{paymentType}/receipts/{orderId}`
 
-## Assisted
 
-Vipps offers assisted content monitoring as a way for Merchants to deal with the regulatory demands of content monitoring. For some merchants Vipps can utilize the merchant's webpage for content monitoring, continuously verifying that the actual products being sold coincides with the expected products.
+# Categories
+* [API Documentation with examples here](https://vippsas.github.io/vipps-order-management-api/redoc.html#tag/Category)
 
-### Mandatory use case
+The following section will explain how to enrich a Vipps transaction with `Categories` and `Images`. A `Link` and `Category` is required when using this API,
+whereas `Images` are optional.
+
+## Category
+In order to provide customers with more up to date information about their order,
+you can add a `Category` with a URL to the Vipps Transaction that can take the
+customer to a location on your website. Links are activated when a customer
+clicks the link in the transaction page in the Vipps App. The mobile device's standard web browser will open and the user is redirected to the link location. Below you can se an example of a Vipps transaction with the "Shipping information" `Category`.
```

Comment:
- "se an example"  -> "see an example"
- I would also change "the user is redirected " to "the user will be redirected " (to match the earlier part of the sentence)